xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Team.xml

```diff
@@ -851,6 +851,15 @@
     #end
   #end
   {{html wiki=false clean=false}}
+  ##Starting with XWiki version 13.10.4 and 14.1-rc-1, img tags have a height: auto set and this breaks the macro fixed size feature. For this, a specific size is added
```

Comment:
You forgot a space after `##`
Same for below, it's `#set ($escap...`

Also, is this line longer than 120 characters? This is the characters limit used